docs/usage/configuration-options.md

```diff
@@ -1652,6 +1652,31 @@ For example to apply a special label for Major updates:
 }
 ```
 
+### replacementName
+
+Use this field to define the name of a replacement package.
+Must be used with `replacementVersion` (see example below).
+New package rules can be added by editing [`replacements.ts`](https://github.com/renovatebot/renovate/blob/main/lib/config/presets/internal/replacements.ts)
```

Comment:
```suggestion
You can suggest a new community package rule by editing [the `replacements.ts` file on the Renovate repository](https://github.com/renovatebot/renovate/blob/main/lib/config/presets/internal/replacements.ts) and opening a pull request.
```

- Clarify these are "community package rules".
- Use active voice for the "adding community rules to repository".
- Improve text for  the clickable link.
- End sentence with a full stop.